pages/national_resources/fi_resources.md

```diff
@@ -2,7 +2,7 @@
 title: Finland
 country_code: FI
 search_exclude: true
```

Comment:
```suggestion
```

If you are happy with the content this line can be deleted so the page appears in the navigation!

Reply:
Looks good to me! Thanks @bedroesb

Reply:
Yes thank you so much!